soh/src/overlays/actors/ovl_Bg_Mori_Hineri/z_bg_mori_hineri.c

```diff
@@ -255,29 +263,6 @@ void BgMoriHineri_DrawHallAndRoom(Actor* thisx, GlobalContext* globalCtx) {
                   G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
         gSPDisplayList(POLY_OPA_DISP++, gDungeonDoorDL);
     }
-    if ((this->boxObjIdx > 0) && ((this->boxObjIdx = Object_GetIndex(&globalCtx->objectCtx, OBJECT_BOX)) > 0) &&
-        Object_IsLoaded(&globalCtx->objectCtx, this->boxObjIdx)) {
-        gSPSegment(POLY_OPA_DISP++, 0x06, globalCtx->objectCtx.status[this->boxObjIdx].segment);
-        gSPSegment(POLY_OPA_DISP++, 0x08, &D_80116280[2]);
-        Matrix_Put(&mtx);
-        Matrix_Translate(147.0f, -245.0f, -453.0f, MTXMODE_APPLY);
-        Matrix_RotateY(M_PI / 2, MTXMODE_APPLY);
-        Matrix_Scale(0.01f, 0.01f, 0.01f, MTXMODE_APPLY);
-        gSPMatrix(POLY_OPA_DISP++, MATRIX_NEWMTX(globalCtx->state.gfxCtx),
-                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-        gSPDisplayList(POLY_OPA_DISP++, gTreasureChestBossKeyChestFrontDL);
-        Matrix_Put(&mtx);
-        Matrix_Translate(167.0f, -218.0f, -453.0f, MTXMODE_APPLY);
-        if (Flags_GetTreasure(globalCtx, 0xE)) {
-            Matrix_RotateZ(0x3500 * (M_PI / 0x8000), MTXMODE_APPLY);
-        } else {
-            Matrix_RotateZ(M_PI, MTXMODE_APPLY);
-        }
-        Matrix_Scale(0.01f, 0.01f, 0.01f, MTXMODE_APPLY);
-        gSPMatrix(POLY_OPA_DISP++, MATRIX_NEWMTX(globalCtx->state.gfxCtx),
-                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-        gSPDisplayList(POLY_OPA_DISP++, gTreasureChestBossKeyChestSideAndTopDL);
-    }
```

Comment:
just want to make sure there isn't anything funky happening in here that switching to using the box actor changes. i get the impression from a player stand point this won't change authentic behavior, but i figure it's good to double check

Reply:
yep, should be no side effects, the only things happening here are graphics calls. From my testing having an extra actor in that room causes no issues either.